Cargo.toml

```diff
@@ -126,7 +126,7 @@ rand = "=0.9.1"
 regex = "1"
 reqwest = { version = "0.12", default-features = false }
 rstest = "0.25.0"
-schemars = "0.8.22"
+schemars = "0.9.0"
```

Comment:
We can't update `schemars` to 0.9.0 because `apollo-federation-types` was compiled with schemars 0.8.22.
https://github.com/apollographql/federation-rs/blob/a8a1d835087502e8a24395b363f8868817b4e59c/apollo-federation-types/Cargo.toml#L29